.github/workflows/integration-tests.yml

```diff
@@ -47,6 +47,10 @@ jobs:
             wp: 'trunk'
             coverage: none
             experimental: false
+          - php: '8.3'
+            wp: 'trunk'
+            coverage: none
+            experimental: false
```

Comment:
- Any changes for line 31, 36 of this file?
- Any changes for `unit-tests.yml` file?

Reply:
- The comments on lines 26-31 are actually outdated, and I plan to correct them in another PR when everything related is merged.
- No changes are needed in line 36 as far as I'm aware, unless I'm missing something.

Reply:
Thanks for spotting that I should also make changes to `unit-tests.yml`, this was an omission from my part.